static/app/components/onboarding/footer.tsx

```diff
@@ -76,22 +68,14 @@ export function Footer({projectSlug, projectId, router, newOrg}: Props) {
   const [firstIssue, setFirstIssue] = useState<Group | undefined>(undefined);
   const [clientState, setClientState] = usePersistedOnboardingState();
   const {projects} = useProjects();
-
-  const onboarding_sessionStorage_key = `onboarding-${projectId}`;
-
-  const [sessionStorage, setSessionStorage] = useSessionStorage<OnboardingState>(
-    onboarding_sessionStorage_key,
-    {
-      status: OnboardingStatus.WAITING,
-      firstIssueId: undefined,
-    }
-  );
+  const onboardingContext = useContext(OnboardingContext);
+  const projectData = projectId ? onboardingContext.data[projectId] : undefined;
 
   useQuery<Project>([`/projects/${organization.slug}/${projectSlug}/`], {
     staleTime: 0,
     refetchInterval: DEFAULT_POLL_INTERVAL,
     enabled:
-      !!projectSlug && !firstError && sessionStorage.status === OnboardingStatus.WAITING, // Fetch only if the project is available and we have not yet received an error,
```

Comment:
@priscilawebdev so we are using a combination of redis state and session storage for onboarding?